diff --git a/paddlex/inference/models/common/static_infer.py b/paddlex/inference/models/common/static_infer.py
index 3b4de05..274e775 100644
--- a/paddlex/inference/models/common/static_infer.py
+++ b/paddlex/inference/models/common/static_infer.py
@@ -404,10 +404,6 @@ class PaddleInfer(StaticInfer):
                 # TODO(changdazhou): use a black list instead
                 if self._model_name == "PP-DocLayoutV3":
                     config.delete_pass("matmul_add_act_fuse_pass")
-                # ROCm does not support fused_conv2d_add_act kernel, delete the fuse passes
-                if paddle.is_compiled_with_rocm():
-                    config.delete_pass("conv2d_add_act_fuse_pass")
-                    config.delete_pass("conv2d_add_fuse_pass")
             elif self._option.device_type == "npu":
                 config.enable_custom_device("npu", self._option.device_id)
                 if hasattr(config, "enable_new_ir"):
@@ -457,11 +453,6 @@ class PaddleInfer(StaticInfer):
                 config.disable_mkldnn()
                 if hasattr(config, "enable_new_executor"):
                     config.enable_new_executor()
-                # XXX: is_compiled_with_rocm() must be True on dcu platform ?
-                if paddle.is_compiled_with_rocm():
-                    # Delete unsupported passes in dcu
-                    config.delete_pass("conv2d_add_act_fuse_pass")
-                    config.delete_pass("conv2d_add_fuse_pass")
             elif self._option.device_type == "iluvatar_gpu":
                 config.enable_custom_device("iluvatar_gpu", int(self._option.device_id))
                 if hasattr(config, "enable_new_ir"):
@@ -486,9 +477,6 @@ class PaddleInfer(StaticInfer):
                 if hasattr(config, "enable_new_executor"):
                     config.enable_new_executor()
                 config.set_optimization_level(3)
-                if paddle.is_compiled_with_rocm():
-                    config.delete_pass("conv2d_add_act_fuse_pass")
-                    config.delete_pass("conv2d_add_fuse_pass")
         config.enable_memory_optim()
         for del_p in self._option.delete_pass:
             config.delete_pass(del_p)
@@ -496,10 +484,6 @@ class PaddleInfer(StaticInfer):
         # Disable paddle inference logging
         if not DEBUG:
             config.disable_glog_info()
-        # ROCm does not support fused_conv2d_add_act kernel, delete the fuse passes
-        if paddle.is_compiled_with_rocm():
-            config.delete_pass("conv2d_add_act_fuse_pass")
-            config.delete_pass("conv2d_add_fuse_pass")
 
         predictor = paddle.inference.create_predictor(config)
 
diff --git a/paddlex/inference/utils/misc.py b/paddlex/inference/utils/misc.py
index 4451098..8c3c5f8 100644
--- a/paddlex/inference/utils/misc.py
+++ b/paddlex/inference/utils/misc.py
@@ -31,7 +31,7 @@ def is_bfloat16_available(device):
     device_type, _ = parse_device(device)
     return (
         "npu" in get_device_type() or paddle.amp.is_bfloat16_supported()
-    ) and device_type in ("gpu", "npu", "xpu", "mlu", "metax_gpu", "iluvatar_gpu")
+    ) and device_type in ("gpu", "dcu", "npu", "xpu", "mlu", "metax_gpu", "iluvatar_gpu")
 
 
 def is_float16_available(device):
